main/realmain.cc

```diff
@@ -531,7 +531,9 @@ int realmain(int argc, char *argv[]) {
             gs->suppressErrorClass(core::errors::Namer::ModuleKindRedefinition.code);
             gs->suppressErrorClass(core::errors::Resolver::StubConstant.code);
 
+            indexed = pipeline::package(*gs, move(indexed), opts, *workers);
```

Comment:
The `pipeline::package` function already handles "do not run if `--stripe-packages` is not supplied," so we don't need to do any conditional logic here.